tests/FondOfSpryker/Yves/EnhancedEcommerceCheckoutConnector/EnhancedEcommerceCheckoutConnectorDependencyProviderTest.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+namespace FondOfSpryker\Yves\EnhancedEcommerceCheckoutConnector;
+
+use Codeception\Test\Unit;
+use Spryker\Yves\Kernel\Container;
+
+class EnhancedEcommerceCheckoutConnectorDependencyProviderTest extends Unit
+{
+    /**
+     * @var \PHPUnit\Framework\MockObject\MockObject|\Spryker\Yves\Kernel\Container
+     */
+    protected $containerMock;
+
+    /**
+     * @var \FondOfSpryker\Yves\GoogleTagManagerQuoteConnector\GoogleTagManagerQuoteConnectorDependencyProvider
+     */
+    protected $dependencyProvider;
+
+    /**
+     * @return void
+     */
+    protected function _before(): void
+    {
+        $this->containerMock = $this->getMockBuilder(Container::class)
+            ->disableOriginalConstructor()
+            ->getMock();
+
+        $this->dependencyProvider = new EnhancedEcommerceCheckoutConnectorDependencyProvider();
+    }
+
+    /**
+     * @return void
+     */
+    public function testProvideDependencies(): void
+    {
+        $this->assertInstanceOf(
+            Container::class,
+            $this->dependencyProvider->provideDependencies(
+                $this->containerMock
+            )
+        );
+    }
+}
```

Comment:
This test is probably unnecessary and doesn't bring any benefit. If it doesn't return a container it would be cought by static code analysis.

Reply:
Yes, I'm wondering myself. Just copy it from the unit tests of DR xD

Should i remove it? Is there a need to test DP? Any ideas to do it better?

Reply:
I would remove it.